API/RestAPI.tsx

```diff
@@ -14,7 +15,7 @@ export async function restFetch(
 
   return fetch(url, {
     headers: {
-      Authorization: `Basic ${appPassword}`,
+      Authorization: `Basic ${Base64.encode(appPassword)}`,
```

Comment:
So now `appPassword` is in format `username:password` as described [here](https://make.wordpress.org/core/2020/11/05/application-passwords-integration-guide/) (REST API).

One thing to discuss is: should we encode the `appPassword` in native apps or here?

I think it'll be better to do in native apps becase:
- we already have to build "app password" to format `username:password`
- we already have to encode to base64 in native apps, so maybe there's no reason to support this 3 times
- if we encode in native apps, we don't have to include a dependency to React Native project

WDYT @Ecarrion ?